data/free worlds start.txt

```diff
@@ -1615,6 +1769,122 @@ mission "FW Pirates 4.1"
 
 
 
+mission "FW Pirates 4.2"
+	landing
+	name "Senate Hearing"
+	description "Report to <destination>, for a Senate hearing on your recent failure to obey their orders. If you do not arrive by <date>, your service will be terminated."
+	source "Longjump"
+	destination "Bourne"
+	deadline 14
+	to offer
+		has "fw pirates: crushed"
+	on fail
+		dialog `You have failed an essential Free Worlds mission. If you want to complete the story line, revert to the autosave or another earlier snapshot of the game.`
+	
+	on offer
+		conversation
+			`JJ and Freya meet you on your ship soon after you land on <origin>, accompanied by two uniformed military policemen. 
+			`	"We heard about the hornet's nest you stirred up from the other captains, before we arrested them," says JJ. "Congratulations. you escaped from the disaster brought on by your stupidity, and the pirates are united against us. Now we have to fight a war on two fronts while you sit in prison with Tomek." He gestures to two military policemen, who pull out handcuffs and move toward you.`
+			choice
+				`	"Are you aware that I destroyed the pirate armada? Let me show you."`
+					goto recording
+				`	You hold out your wrists. "I will submit to your authority."`
+					goto arrested
+				`	You surreptitiously flip on the fleet broadcaster. "You're arresting me? For what?"`
```

Comment:
Would be more consistent to write action-choices like [this:](https://github.com/endless-sky/endless-sky/blob/master/data/transport%20missions.txt#L147)
> 		choice
			 `	(Try the service tunnel.)`
				goto tunnel
			`	(Wait.)`
				goto outside
			` "I'm sorry, this isn't worth risking my life for. I'm leaving."`

Github is weird.